.github/workflows/prod-deploy.yml

```diff
@@ -0,0 +1,92 @@
+name: Deploy to Production
+
+on:
+  workflow_run:
+    workflows: ["Production — Push container to ECR"]
+    types:
+      - completed
+
+jobs:
+  unable-to-deploy:
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify Slack if ECR build/push failed
+        env:
+          WORKFLOW_URL: "${{ github.event.workflow_run.html_url }}"
+          WORKFLOW_NAME: "${{ github.event.workflow_run.name }}"
+        run: |
+          json='{"channel":"#forms-production-events", "blocks":[{"type":"section","text":{"type":"mrkdwn","text":":red: GitHub workflow failure: <${{ env.WORKFLOW_URL }}|${{ env.WORKFLOW_NAME }}>"}}]}'
+          curl -X POST -H 'Content-type: application/json' --data "$json" ${{ secrets.PRODUCTION_SLACK_WEBHOOK }}
+          exit 1   
+
+  deploy-form-viewer-service:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2.7.0
+
+      - name: Configure AWS credentials
+        # v1 as of Jan 28 2021
+        uses: aws-actions/configure-aws-credentials@8c3f20df09ac63af7b3ae3d7c91f105f857d8497
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ca-central-1
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        # v1 as of Jan 28 2021
+        uses: aws-actions/amazon-ecr-login@865664b3240e20b35d3528593d496632704f48af
+
+      - name: Download Form Viewer task definition
+        id: download-taskdef-form-viewer
+        run: |
+          aws ecs describe-task-definition --task-definition form-viewer --query taskDefinition > form_viewer.json
+          echo "container_name=$(jq -r '.containerDefinitions[0].name' form_viewer.json)" >> "$GITHUB_OUTPUT"
+
+      - name: Render image for form viewer service
+        id: taskdef-form-viewer
+        # v1.0.10
+        uses: aws-actions/amazon-ecs-render-task-definition@c6f3dfbdd9990283e47c9e0cf46393ea13f1541b
+        with:
+          task-definition: form_viewer.json
+          container-name: ${{ steps.download-taskdef-form-viewer.outputs.container_name }}
+          image: ${{ steps.login-ecr.outputs.registry }}/form_viewer_production:${{ github.event.workflow_run.head_branch }}
```

Comment:
Not sure if `github.event.workflow_run.head_branch` will give the tag version number but willing to try it out.

Reply:
I was skeptical as well based on the name and total lack of meaningful docs 🙄  so I ran a bunch of tests in a personal repo.